tools/dependencies/known-dependencies.txt

```diff
@@ -558,7 +558,9 @@ metrics-core-3.1.5.jar
 metrics-core-3.2.1.jar
 metrics-core-4.0.0.jar
 metrics-json-3.1.0.jar
+metrics-json-3.1.5.jar
 metrics-jvm-3.1.0.jar
+metrics-jvm-3.1.5.jar
```

Comment:
We'd better unify the version to facilitate subsequent maintenance. I've been doing something related recently.

Reply:
You have a lot of unnecessary formatting changes, please revert it.